hamlet-cli/hamlet/command/schema/__init__.py

```diff
@@ -0,0 +1,224 @@
+import os
+import re
+import click
+
+from tabulate import tabulate
+
+from hamlet.command import root as cli
+from hamlet.command.common.display import json_or_table_option, wrap_text
+from hamlet.command.common.exceptions import CommandError
+from hamlet.backend import query as query_backend
+from hamlet.backend.create import template as create_template_backend
+from hamlet.backend.manage import stack as manage_stack_backend
+from hamlet.backend.manage import deployment as manage_deployment_backend
+from hamlet.backend.common.exceptions import BackendException
+
+def find_schemas_from_options(generation, deployment_group, deployment_units):
+    query_args = {
+        'deployment_mode': '',
+        'generation_entrance': 'schemaset',
+        'generation_input_source': generation.generation_input_source,
+        'generation_provider': generation.generation_provider,
+        'generation_framework': generation.generation_framework,
+        'output_filename': 'schemaset-' + deployment_group + '-schemacontract.json',
+        'refresh_output': True
+    }
+    try:
+        available_schemas = query_backend.run(
+            **query_args,
+            cwd=os.getcwd(),
+            query_text=LIST_SCHEMAS_QUERY
+        )
+
+    except BackendException as e:
+        raise CommandError(str(e))
+
+    deployments = []
```

Comment:
Might want to rename that as its not specifcally a deployment

Reply:
Corrected, thanks